### PR TITLE
Update rootless docker images to the latest

### PR DESCRIPTION
**Pull request by phuongnq**
https://github.com/craftercms/craftercms/issues/6621

### CODE REVIEW

**Review by avasquez614 — CHANGES_REQUESTED** [review]
@phuongnq I think we need a rootless brach for 4.1 too

**Comment by phuongnq** [comment]
@avasquez614 This is actually for 4.1.x. I have updated the default flag to 4.1.4
As of current, we don't have any difference between 4.1 and develop branch. When we have the new Spring 6 PRs merge, it could be changed though (since Spring 6 uses Tomcat 10)

File: images/authoring_local/tomcat/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ FROM eclipse-temurin:17-jdk-alpine AS main
 ############# INSTALL NATIVE TOMCAT #############
 ENV CATALINA_HOME=/usr/local/tomcat \
     TOMCAT_MAJOR="9" \
-    TOMCAT_VERSION="9.0.80"
+    TOMCAT_VERSION="9.0.85"
```
Comment (sumerjabri) [inline images/authoring_local/tomcat/Dockerfile]: 9.0.86?

https://github.com/craftercms/craftercms/blob/support/4.1.x/pom.xml#L187

File: images/delivery/tomcat/Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ FROM eclipse-temurin:17-jdk-alpine AS main
 ############# INSTALL NATIVE TOMCAT #############
 ENV CATALINA_HOME=/usr/local/tomcat \
     TOMCAT_MAJOR="9" \
-    TOMCAT_VERSION="9.0.80"
+    TOMCAT_VERSION="9.0.85"
```
Comment (sumerjabri) [inline images/delivery/tomcat/Dockerfile]: .86

**Review by sumerjabri — CHANGES_REQUESTED** [review]
I updated the base to use Tomcat 9.0.86, please do the same.

https://github.com/craftercms/docker-images/pull/90/files

File: images/authoring_local/tomcat/Dockerfile (hunk 1 above)
Comment (phuongnq) [reply images/authoring_local/tomcat/Dockerfile]: Updated

File: images/delivery/tomcat/Dockerfile (hunk 2 above)
Comment (phuongnq) [reply images/delivery/tomcat/Dockerfile]: Updated

**Review by avasquez614 — APPROVED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**